Dockerfile

```
@@ -1,4 +1,90 @@
-FROM ghcr.io/geniusyield/haskell-base-image:9.2.8
+FROM debian:bullseye-slim
```

Please use bookworm-

```suggestion
FROM debian:bookworm-slim
```

Why now build from our base image?

---

Since we realise there is also ghcr.io/geniusyield/haskell-base-image:9.6.5, I have updated to it. Let's see if it builds correctly.